README.md

```diff
@@ -59,6 +59,19 @@ import { FlatGrid } from 'react-native-super-grid';
 ```
 
 
+### Usage (SimpleGrid)
+
+```javascript
```

Comment:
A one liner description of what SimpleGrid is would be nice.

Reply:
done in [268cdb9](https://github.com/saleel/react-native-super-grid/pull/196/commits/268cdb985e7552594a2cc671eff4ef76df2bd39c)